packages/forma-36-react-components/tools/plop-templates/components/readme.mdx.hbs

```diff
@@ -1,23 +1,33 @@
 # {{pascalCase name}}
 
-A short summary of the component.
+* When to use the component, the challenges it can solve.
+* When to consider using another component (when there might be confusion).
+* (optional) What it consists of (if it's composed of other components).
 
-## Variations/types
+## Table of contents
 
-Provide the overview of the components variants.
+- [How to use {{pascalCase name}}](#how-to-use-{{pascalCase name}})
+- [Component variations](#component-variations)
+- [Code examples](#code-examples)
+- [Content recommendations](#content-recommendations)
+- [Accessibility](#accessibility)
 
-## Example of usage
+## How to use {{pascalCase name}}
 
-Give couple of clear examples of the usage of the components, that includes import
+Consider common mistakes, positioning, interaction with other elements and best practices.
 
-## Content recommendations
+## Component variations
+
+Provide an overview of all the variations.
 
-Add notes about best practices when it comes to the content used in the component, like message or title
+## Code examples
 
-## Best practices
+Give at least one code example, starting from the most basic version.
```

Comment:
I would suggest adding that it should include import.

Reply:
We just learned though that it has to be in the comment, for it to work properly on the website.